### ISSUE TITLE
adding danish hunspell dictionaries leads to hilariously many false positives

### ISSUE THREAD

**Issue by norseghost** [jdkato/txtlint#1065]
### Check for existing issues

- [x] Completed

### Environment

windows, installed via winget, vale version 3.13

### Describe the bug / provide steps to reproduce it

using a style guide to load danish spell check (dic and aff files from stavekontrolden.dk) , I get very very many obvious false positives. 

on the level of "this is a sentence with words I have spelled correctly" flagging every word except "thtis" and "a". 

hunspell ran directly actually flagged the one actual mistake I made.

dictionaries in stylesdir/config/directories
my rule:

```yaml
extends: spelling
level: error
message: "'%s' er ikke dansk"
dictionaries:
  - da_DK
 ```

**Comment by norseghost** [comment]
[rapunzel-da.txt](https://github.com/user-attachments/files/24553394/rapunzel-da.txt) 

this danish version of Rapunzel can be used as a test document. 

Straight hunspell flags 3 errors: one misspelling (hvergang instead of hver gang) and two nonsense fairy tale words (ritsch ratsch). 

vale thinks 292 out of 1220 words are misspelled (they are not): 

[vale.log](https://github.com/user-attachments/files/24553435/vale.log)

**Comment by jdkato** [comment]
This should be fixed in the next release. The issue was that the dic contains a malformed entry — `coituum` and `/34 st:coitus` split across two lines, preventing it from being loaded at all.
